### PR TITLE
fix(domain.dnsserver): add-icon-external-link

### PR DESCRIPTION
**Pull request by TedKac**

### CODE REVIEW

**Comment by frenautvh** [comment]
i mean remove the link from the translation and only translate the sentence

**Comment by TedKac** [comment]
@frenautvh requested changes has been made :)

File: client/app/domain/dns/DNS.html [hunk 1]
```diff
@@ -17,8 +17,12 @@
                  data-ng-if="ctrlDomainDns.dns.activeDns === 1 && !ctrlDomainDns.dns.table.isPartial"></div>
 
             <div class="alert alert-info" role="alert"
-                 data-ng-bind-html="tr('domain_tab_DNS_unlock_warning_manuel_check', [ctrlDomainDns.urls.zoneCheck, i18n.domain_tab_DNS_unlock_warning_manuel_check_tool + ' (' + i18n.core_new_window + ')'])"
                  data-ng-if="ctrlDomainDns.allowModification">
+                 <span data-ng-bind="tr('domain_tab_DNS_unlock_warning_manuel_check')"></span>
+                 <a data-ng-href="{{ctrlDomainDns.urls.zoneCheck}}" title="{{i18n.domain_tab_DNS_unlock_warning_manuel_check_tool + ' (' + i18n.core_new_window + ')'}}" target="_blank" >
```
Comment (antleblanc) [inline client/app/domain/dns/DNS.html]: You can remove the extra space `target="_blank" >`

Comment (TedKac) [reply client/app/domain/dns/DNS.html]: Done :)

**Pull request merged**